clients/banking/src/components/AccountMembershipArea.tsx

```diff
@@ -33,30 +33,41 @@ export const AccountMembershipArea = ({ accountMembershipId }: Props) => {
   const [updateAccountLanguage] = useMutation(UpdateAccountLanguageDocument);
 
   useEffect(() => {
-    const request = query({ accountMembershipId }).tapOk(({ accountMembership, user }) => {
-      if (accountMembership?.user?.id !== user?.id) {
```

Comment:
wouldn't simply moving the condition to `accountMembership != null && accountMembership?.user?.id !== user?.id` do the trick?